src/integrationTest/java/uk/gov/hmcts/darts/event/controller/DarNotifyControllerTest.java

```diff
@@ -56,14 +68,50 @@ void setup() {
 
     @Test
     void shouldSendDarNotifyEventSoapAction() throws Exception {
-        darPcStub.respondWithSuccessResponse();
+        darPcStub.respondWithSuccessResponse(OffsetDateTime.now(clock));
+
+        mockMvc.perform(post("/events/dar-notify")
+                            .contentType(APPLICATION_JSON_VALUE)
+                            .content(VALID_NOTIFICATION_JSON))
+            .andExpect(status().is2xxSuccessful());
+
+        darPcStub.verifyNotificationReceivedWithBody(EXPECTED_DAR_PC_NOTIFICATION);
+    }
+
+    @Test
+    @ExtendWith(OutputCaptureExtension.class)
+    void shouldSendDarNotifyEventSoapActionDarPcDateOutSide2MinRangeBehind(CapturedOutput capturedOutput) throws Exception {
+        OffsetDateTime responseDateTime = OffsetDateTime.now(clock).minusMinutes(2).truncatedTo(ChronoUnit.SECONDS);
+        darPcStub.respondWithSuccessResponse(responseDateTime);
 
         mockMvc.perform(post("/events/dar-notify")
                             .contentType(APPLICATION_JSON_VALUE)
                             .content(VALID_NOTIFICATION_JSON))
             .andExpect(status().is2xxSuccessful());
 
         darPcStub.verifyNotificationReceivedWithBody(EXPECTED_DAR_PC_NOTIFICATION);
+        assertThat(capturedOutput)
+            .containsPattern("Response time from DAR PC is not within 2 minutes of current time. DarPC Response time: "
+                                 + responseDateTime.format(DateTimeFormatter.ISO_DATE_TIME)
+                                 + ", Current time: [0-9\\-.T:]+Z for url: http://localhost:8090/VIQDARNotifyEvent/DARNotifyEvent.asmx");
+    }
+
+    @Test
+    @ExtendWith(OutputCaptureExtension.class)
+    void shouldSendDarNotifyEventSoapActionDarPcDateOutSide2MinRangeAhead(CapturedOutput capturedOutput) throws Exception {
+        OffsetDateTime responseDateTime = OffsetDateTime.now().plusMinutes(2).truncatedTo(ChronoUnit.SECONDS);
```

Comment:
Should that be `OffsetDateTime.now(clock)...`?

Reply:
Yes good spot